drivers/mtd/Make.defs

```diff
@@ -29,6 +29,10 @@ ifeq ($(CONFIG_MTD),y)
 
 CSRCS += ftl.c mtd_config.c
 
+ifeq ($(CONFIG_MTD_NVS),y)
+CSRCS += mtd_nvs.c
+endif
```

Comment:
ifeq ($(CONFIG_MTD_NVS),y)
CSRCS += mtd_nvs.c
else if($(CONFIG_MTD_CONFIG),y)
CSRCS += mtd_config.c
endif

Reply:
mtd_config.c was added in line 30 of this file. So this is not necessary

Reply:
After some thinking, I will change it.